src/testdata/testwallets/multipleValidatorKeys.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PRIVATE KEY for f8910e47cf9464777c912e6390758bb39715fffcb861b184017920e4a807b42553f2f21e7f3914b81bcf58b66a72ab16d97013ae1cff807cefc977ef8cbf116258534b9e46d19528042d16ef8374404a89b184e0a4ee18c77c49e454d04eae8d-----
```

Comment:
All good, not included in the NPM package:

https://github.com/multiversx/mx-sdk-js-core/blob/main/.npmignore